database-test-data/UserSchema-data-set.json

```diff
@@ -0,0 +1 @@
+{"name": "tim the terminator", "username":"lil timmy", "pass":"testPass123$", "email":"tim_rules@fake.com", "subscriptions":[], "currentId": 10138 }
```

Comment:
can't wait to try out this data set. Thank you for creating these fake data objects